ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleExtrasBuilder.java

```diff
@@ -48,11 +47,6 @@ public VungleExtrasBuilder setUserId(String userId) {
     return this;
   }
 
-  public VungleExtrasBuilder setFlexViewCloseTimeInSec(int flexViewCloseTimeInSec) {
```

Comment:
is it ok to just remove this API instead of deprecation at least for 1 release or make it do nothing?